### PR TITLE
Uctit sd 01 002 hero

### PR DESCRIPTION
**Pull request by jobayeructit**
[hero.webm](https://github.com/UCT-IT/software-development/assets/149924386/b5cdd52b-8904-4135-af1f-fd4994aa325e)
hero

### CODE REVIEW

File: src/app/assets/locales/locales.ts [hunk 1]
```diff
@@ -2,7 +2,24 @@ import LocalizedStrings from "react-localization";
...
+          text2: "Digital Potential",
+        },
+        description: `Your one-stop digital software solutions company. At Tech, we thrive
+         on transforming your ideas into cutting-edge digital realities.`,
+         buttons:{
+          button1: "start a project",
+          button2: "out work"
+         },
+         altText: {
+          logo: "hero icon",
```
Comment (saminsss) [inline src/app/assets/locales/locales.ts]: Can you make the image alt more descriptive?

File: src/app/assets/locales/locales.ts [hunk 2]
```diff
@@ -2,7 +2,24 @@ import LocalizedStrings from "react-localization";
...
+          gallry: "gallery image"
```
Comment (saminsss) [inline src/app/assets/locales/locales.ts]: Typo here `gallry`

File: src/app/globals.css [hunk 3]
```diff
@@ -2,3 +2,13 @@
 @tailwind components;
 @tailwind utilities;
 
+.responsive-container {
+  @apply container mx-auto px-5 md:px-10 lg:px-5;
+}
+.hero-bg {
+  background: url("../../public/images/home/hero/heroBg.svg");
```
Comment (saminsss) [inline src/app/globals.css]: Can you add bg image without CSS?

File: src/app/assets/locales/locales.ts [hunk 4]
```diff
@@ -2,7 +2,24 @@ import LocalizedStrings from "react-localization";
...
+      hero: {
+        title: {
+          text1: "Unlock Your",
+          text2: "Digital Potential",
+        },
+        description: `Your one-stop digital software solutions company. At Tech, we thrive
+         on transforming your ideas into cutting-edge digital realities.`,
+         buttons:{
+          button1: "start a project",
+          button2: "out work"
```
Comment (saminsss) [inline src/app/assets/locales/locales.ts]: Typo

**Review by saminsss — CHANGES_REQUESTED** [review]

File: src/app/assets/locales/locales.ts (hunk 4 above)
Comment (jobayeructit) [reply src/app/assets/locales/locales.ts]: done

File: src/app/globals.css (hunk 3 above)
Comment (jobayeructit) [reply src/app/globals.css]: done

File: src/app/assets/locales/locales.ts (hunk 2 above)
Comment (jobayeructit) [reply src/app/assets/locales/locales.ts]: done

File: src/app/assets/locales/locales.ts (hunk 1 above)
Comment (jobayeructit) [reply src/app/assets/locales/locales.ts]: done

**Review by saminsss — APPROVED** [review]

**Pull request merged**